### PR TITLE
PD-1572 / 25.04 / Pd 1572 review ipmi configuration tutorial

### PR DESCRIPTION
**Pull request by DjP-iX**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Network/SettingUpIPMI.md [hunk 1]
```diff
@@ -7,32 +7,53 @@ tags:
 - ipmi
 ---
 
-{{< hint type=note >}}
-IPMI requires compatible hardware! Refer to your hardware documentation to determine if the TrueNAS web interface has IPMI options.
-{{< /hint >}}
+{{< include file="/static/includes/IPMICompatibility.md" >}}
 
-Many [TrueNAS Storage Arrays](https://www.truenas.com/docs/hardware/) have a built-in out-of-band management port that provides side-band management should the system become unavailable through the web interface.
+Many [TrueNAS systems](https://www.truenas.com/docs/hardware/) have a built-in out-of-band management port that provides side-band management should the system become unavailable through the web interface.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/Network/SettingUpIPMI.md:12]: I think this sentence, though well written, might be hard to parse or translate for ESL readers. That said, I don't know how to rephrase this to make it easier. Thoughts?

Comment (DjP-iX) [reply content/SCALE/SCALETutorials/Network/SettingUpIPMI.md:12]: How about 
```suggestion
Many [TrueNAS systems](https://www.truenas.com/docs/hardware/) include a built-in out-of-band management port, enabling system access even when the web interface is unavailable.
```

**Review by micjohnson777 — APPROVED** [review]
